docs/adr/CONTRIBUTING.md

```diff
@@ -0,0 +1,23 @@
+# What is this?
+Architecture Decision Records (ADR's) are a way to capture abstract choices that engineers make throughout a repo. Whenever the Stencil team discovers a concept that's hard to share with other people in plain language, that's often when we'll find an opportunity to write an ADR. 
+
+These ADR's serve a few different people:
+1. It helps the Stencil team to communicate together about a decision that could make an impact, at any point during our process. 
+2. It helps Community members understand where we're at on decisions. 
+3. It helps the Stencil team begin to have open discussion about the changes we're making to Stencil. 
+4. It helps new developers, whether they're on the Stencil team or they're in the community, learn about the decisions we've made within the code.
+5. It helps eveyone to have a running, historical list of changes that have been incorporated into the codebase. 
+
+# Who can propose an ADR? 
+Anybody can propose an ADR and request for comment on them. 
+
+
+# How do I contribute? 
+We use the [Node version of ADR tools](https://github.com/phodal/adr), installable via npm as:
+```
+npm install -g adr
```

Comment:
How did we land on this tool? It looks like there's a few different ones out there:
- [adr-log](https://adr.github.io/adr-log/)
- [adr-tools](https://www.npmjs.com/package/adr-tools)

To be fair, I haven't dug into these in depth - this is just from a cursory Google search.  I was out Friday, so I'm just making sure we've done our due diligence here, sorry if I missed it in Slack/elsewhere!

Reply:
`adr-tools` is a fork of this `adr` package. `adr-log` doesn't have as many features as the `adr` package. Finally, this `adr` npm package is listed as the node rewrite of the original `adr-tools` [here](https://adr.github.io/#tooling), which would need to be installed via brew for us - I decided to use the node rewrite for platform independence.